kiwi/system/kernel.py

```diff
@@ -21,6 +21,8 @@
 # project
 from ..command import Command
 
+from ..exceptions import KiwiKernelLookupError
```

Comment:
I am honestly still not fan of those relative imports. It somehow needs to be done like:
```python
from full.path.to.the.module import myfunction, myclass
```
But this is a next PR, I guess.